app/code/Mykhailok/SupportChat/Controller/Message/Save.php

```diff
@@ -122,8 +122,7 @@ public function execute()
             $this->messageManager->addErrorMessage(__('Unable to connect to the store support team.'));
         }
 
-        return $result
-            ->setData($this->responseData->getData());
+        return $result->setData($this->responseData->getData());
```

Comment:
@mykhailok-dv Controller must not return messages. Saving messages triggers customerData reload. This is handled by JS.Controller can just return empty response.

Reply:
The unreadable messages variable was removed